src/script/view_model/CallingViewModel.mocks.ts

```diff
@@ -28,17 +28,20 @@ import {CallingViewModel} from './CallingViewModel';
 import {Call} from '../calling/Call';
 import {CallingRepository} from '../calling/CallingRepository';
 import {CallState} from '../calling/CallState';
+import {Conversation} from '../entity/Conversation';
 import {Core} from '../service/CoreSingleton';
 
+let callClosedCallback: (conversationId: QualifiedId) => void = (conversationId: QualifiedId) => {};
+
 export const mockCallingRepository = {
   startCall: jest.fn(),
   answerCall: jest.fn(),
-  leaveCall: jest.fn(),
   onIncomingCall: jest.fn(),
   onRequestClientsCallback: jest.fn(),
   onRequestNewEpochCallback: jest.fn(),
   onCallParticipantChangedCallback: jest.fn(),
-  onLeaveCall: jest.fn(),
+  onCallClosed: jest.fn().mockImplementation(callback => (callClosedCallback = callback)),
+  leaveCall: jest.fn().mockImplementation(conversationId => callClosedCallback(conversationId)),
```

Comment:
As we're mocking `CallingRepository` there's no way we could trigger `callClosed` callback from AVS. For the sake of test, we mock `leaveCall` to call our callback immediately, so we're able to test unsubscribing from `newEpoch` mls service event.